runtime/src/accounts_db.rs

```diff
@@ -12721,7 +12720,7 @@ pub mod tests {
 
     #[test]
     fn test_verify_bank_hash_bad_account_hash() {
```

Comment:
```suggestion
    fn test_verify_accounts_hash_bad_account_hash() {
```

Reply:
Looks like several other tests also are no longer accurately named. Missed that in the previous PR. Feel free to do this/those in a separate PR as well.